packages/playground/website/src/components/ensure-playground-site/ensure-playground-site-is-selected.tsx

```diff
@@ -168,9 +168,15 @@ async function createNewTemporarySite(
 	// Lean on the Query API parameters and the Blueprint API to
 	// create the new site.
 	const newUrl = new URL(window.location.href);
+	const defaultBlueprint =
```

Comment:
We'll need to bring that over to the Blueprints gallery: https://github.com/WordPress/blueprints

Reply:
yay, thank you for moving it!